crates/extensions/c8y_mapper_ext/src/actor.rs

```diff
@@ -67,6 +84,7 @@ pub struct C8yMapperActor {
     mqtt_publisher: LoggingSender<MqttMessage>,
     timer_sender: LoggingSender<SyncStart>,
     bridge_status_messages: SimpleMessageBox<MqttMessage, MqttMessage>,
+    message_handlers: HashMap<ChannelFilter, Vec<LoggingSender<MqttMessage>>>,
```

Comment:
Nice!

I really like how this message source can be used to dispatch MQTT messages to subscribers, including the option for a given subscriber to receive message from several channels.